super_sim.go

```diff
@@ -0,0 +1,233 @@
+	AccountSid              string     `json:"account_sid"`
+	UniqueName              string     `json:"unique_name""`
+	DataEnabled             bool       `json:"data_enabled"`
+	DataLimit               int        `json:"data_limit"`
+	DataMetering            string     `json:"data_metering"`
+	DateCreated             TwilioTime `json:"date_created"`
+	DateUpdated             TwilioTime `json:"date_updated"`
+	CommandsEnabled         bool       `json:"commands_enabled"`
+	CommandsUrl             url.URL    `json:"commands_url"`
+	CommandsMethod          string     `json:"commands_method"`
+	SmsCommandsEnabled      bool       `json:"sms_commands_enabled"`
+	SmsCommandsMethod       string     `json:"sms_commands_method"`
+	IPCommandsMethod        string     `json:"ip_commands_method"`
+	IPCommandsUrl           string     `json:"ip_commands_url"`
+	NetworkAccessProfileSid string     `json:"network_access_profile_sid"`
+}
+
+// FleetPage represents a page of Fleets.
+type FleetPage struct {
+	Meta   Meta     `json:"meta"`
+	Fleets []*Fleet `json:"fleets"`
+}
+
+type fleetPageIterator struct {
+	p *PageIterator
+}
+
+// Create creates a new SuperSim Fleet with the data provided, or returns an error.
+func (s *SuperSimService) CreateFleet(ctx context.Context, data url.Values) (*Fleet, error) {
+	fleet := new(Fleet)
+	err := s.client.CreateResource(ctx, fleetPathPart, data, fleet)
+	return fleet, err
+}
+
+// Get finds a single SuperSim Fleet by its sid, or returns an error.
+func (s *SuperSimService) GetFleet(ctx context.Context, sid string) (*Fleet, error) {
+	fleet := new(Fleet)
+	err := s.client.GetResource(ctx, fleetPathPart, sid, fleet)
+	return fleet, err
+}
+
+// GetPage returns a single Page of fleets, filtered by data.
+//
+// See https://www.twilio.com/docs/iot/supersim/api/fleet-resource#read-multiple-fleet-resources.
+func (s *SuperSimService) GetFleetPage(ctx context.Context, data url.Values) (*FleetPage, error) {
+	return s.GetFleetPageIterator(data).Next(ctx)
+}
+
+// GetFleetPageIterator returns a fleetPageIterator with the given page
+// filters. Call iterator.Next() to get the first page of resources (and again
+// to retrieve subsequent pages).
+func (s *SuperSimService) GetFleetPageIterator(data url.Values) *fleetPageIterator {
+	iter := NewPageIterator(s.client, data, fleetPathPart)
+	return &fleetPageIterator{
+		p: iter,
+	}
+}
+
+// Next returns the next page of resources. If there are no more resources,
+// NoMoreResults is returned.
+func (s *fleetPageIterator) Next(ctx context.Context) (*FleetPage, error) {
+	ap := new(FleetPage)
+	err := s.p.Next(ctx, ap)
+	if err != nil {
+		return nil, err
+	}
+	s.p.SetNextPageURI(ap.Meta.NextPageURL)
+	return ap, nil
+}
+
+type UsageRecord struct {
+	AccountSid   string      `json:"account_sid"`
+	SimSid       string      `json:"sim_sid"`
+	FleetSid     string      `json:"fleet_sid"`
+	NetworkSid   string      `json:"network_sid"`
+	DataUpload   int         `json:"data_upload"`
```

Comment:
Might as well make this int64 just to be safe, and download and total. I wish that Twilio included the units in the API response

Reply:
If you know the units, can you add it in a comment?